service/src/test/java/uk/gov/hmcts/reform/fpl/service/config/LookupTestConfig.java

```diff
@@ -8,6 +8,6 @@
 public class LookupTestConfig {
     @Bean
     public HmctsCourtLookupConfiguration courtLookupConfiguration() {
-        return new HmctsCourtLookupConfiguration("example=>Family Court:admin@family-court.com");
+        return new HmctsCourtLookupConfiguration("example=>Family Court:admin@family-court.com:11");
```

Comment:
Can I add that change to prod without pushing the code? Or it's going to cause problems I would like to avoid forgetting.

Reply:
yes, you can. We made an early pr for this use case [Pr-736](https://github.com/hmcts/fpl-ccd-configuration/pull/736)